pkg/config/config.go

```diff
@@ -63,14 +74,31 @@ func Restore(config *api.Config, cmd *cobra.Command) {
 		glog.V(1).Infof("Unable to parse %s: %v", DefaultConfigPath, err)
 		return
 	}
+
+	envOverride := false
 	config.BuilderImage = c.BuilderImage
 	config.Source = c.Source
 	config.Tag = c.Tag
+	cmd.Flags().Visit(func(f *pflag.Flag) {
+		if f.Name == "env" {
+			envOverride = true
+		}
+	})
```

Comment:
can't just use a cmd.Flag("env").Changed check for this?  because it's an array flag?